packages/react-components/react-popover/src/components/Popover/usePopover.ts

```diff
@@ -121,7 +121,11 @@ export const usePopover_unstable = (props: PopoverProps): PopoverState => {
 
   React.useEffect(() => {
     if (open && positioningRefs.contentRef.current) {
-      const firstFocusable = findFirstFocusable(positioningRefs.contentRef.current);
+      const containerTabIndex = positioningRefs.contentRef.current.getAttribute('tabIndex');
```

Comment:
Please add a VR test that verify that the scenario this is for 'avoiding scrolling' does not regress